### DISCUSSION TITLE
slash command permissions

### DISCUSSION THREAD

**Discussion by LewiDev** [discordjs/discord.js#6522] [Q&A] [answered]

**Comment by Jayy** [comment]
Realistically, you could consider using `<interaction>.member` to check permissions and `<interaction>.inGuild()` to ensure the interaction is in a guild. 

Hope this helped!

**Comment by PalmDevs** [comment, marked as answer]
**API limitations**
You cannot set permission bitfield/flags (such as **BAN_MEMBERS**) to the slash commands at the moment.
You can set for permissions `USER` or `ROLE` though!

If you want to work on bitfield and flags, I recommend checking it inside of the code and send an ephemeral message if someone is missing permissions.
